app/src/main/java/org/oppia/android/app/player/state/SelectionInteractionView.kt

```diff
@@ -118,10 +118,6 @@ class SelectionInteractionView @JvmOverloads constructor(
   }
 }
 
-fun setAllOptionsItemInputType(selectionItemInputType: SelectionItemInputType) {
```

Comment:
Random unused function you noticed? Just trying to understand why this was deleted.

Reply:
Yup -- [line 54](https://github.com/oppia/oppia-android/pull/2246/files/7dd9fb0c5eee86fcf98cf777d8441e1b46721b20?file-filters%5B%5D=.kt&file-filters%5B%5D=.xml#diff-bf1ed4362855328c6abbab0f7bc4c152a165b50ddd979209a24839ed5d8b44a6L54) is the exact same function call